src/services/offline/OfflineRecipeService.ts

```
@@ -0,0 +1,725 @@
+          await ApiService.recipes.delete(operation.recipeId);
+        } catch (error) {
+          // If recipe is already deleted on server (404), consider it success
+          if ((error as any).response?.status !== 404) {
+            throw error;
+          }
+        }
+
+        // Remove from offline cache if still there
+        const recipeIndex = state.recipes.findIndex(
+          r => r.id === operation.recipeId
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)